wled00/ir.cpp

```diff
@@ -654,9 +661,7 @@ void handleIR()
       {
         if (results.value != 0) // only print results if anything is received ( != 0 )
         {
-          Serial.print("IR recv\r\n0x");
-          Serial.println((uint32_t)results.value, HEX);
-          Serial.println();
+          DEBUG_PRINTF("IR recv: 0x%lX\n", (unsigned long)results.value);
```

Comment:
This should probably stay `Serial`, so normal users can identify key codes of custom remotes to make an ir.json config, without having to build a debug version.
Should check for TX pin (GPIO1) allocation though, otherwise it could mess with LEDs or other HW connected on the pin.

Reply:
Exactly, as it happened recently on some other issue posted.
It may be better to add this info somewhere else (info page i.e. last IR received).